tests/table-operations.spec.ts

```diff
@@ -490,3 +493,117 @@ test('rollup rows and aggregrate columns', async ({ page }) => {
   });
 });
+
+test('advanced settings', async ({ page }) => {
+  const advancedSettingsOpt = page.locator(
+    'data-testid=menu-item-Advanced Settings'
+  );
+  await advancedSettingsOpt.click();
+  await expect(page.getByText('Table Options')).toHaveCount(0);
+
+  await test.step('toggle settings', async () => {
+    await page
+      .getByTestId(
+        'menu-item-Clear current table filters before applying new filters from a control'
+      )
+      .click();
+    await page
+      .getByTestId(
+        'menu-item-Clear current table filters before applying new filters from an incoming link filter'
+      )
+      .click();
+    await page.getByTestId('btn-page-back').click();
+  });
+
+  await test.step('create input filter', async () => {
+    await page.getByRole('button', { name: 'Controls' }).click();
+
+    const inputFilter = page.getByRole('button', { name: 'Input Filter' });
+    const target = page.getByText('Command History');
+    const dropIndicator = page.locator('.lm_dragProxy');
+    await dragComponent(inputFilter, target, dropIndicator);
+
+    await page.getByTestId('btn-page-close').click();
+  });
+
+  await test.step('toggle quick filter', async () => {
+    const stringColumn = page.locator('.iris-grid .grid-wrapper');
+
+    await stringColumn.click({ position: { x: 20, y: 10 }, button: 'right' });
+    await page.getByRole('button', { name: 'Quick Filters' }).click();
+
+    await page.keyboard.type('null');
+    await waitForLoadingDone(page);
+
+    await expect(page.locator('.iris-grid-column')).toHaveScreenshot();
+  });
+
+  await test.step('use input filter', async () => {
+    const stringColumnOpt = '10';
+    await page.getByRole('combobox').selectOption(stringColumnOpt);
+    await page.getByRole('button', { name: 'Save' }).click();
+
+    await expect(page.getByPlaceholder('Enter value...')).toHaveCount(1);
+    await page.getByPlaceholder('Enter value...').fill('~a9');
+    await page.getByPlaceholder('Enter value...').click();
+    // await page.keyboard.type('~a9');
+    await page.keyboard.press('Enter');
+
+    await waitForLoadingDone(page);
+
+    // await expect(page.locator('.input-filter-value-card')).toHaveScreenshot();
+    await expect(page.locator('.iris-grid-column')).toHaveScreenshot();
+  });
+
+  await test.step('create 2nd table', async () => {
+    const consoleInput = page.locator('.console-input');
+    await consoleInput.click();
+
+    const command = makeTableCommand(undefined, TableTypes.AllTypes);
+
+    await pasteInMonaco(consoleInput, command);
+    await page.keyboard.press('Enter');
+  });
+
+  await test.step('drag table to new panel', async () => {
+    // opening up this menu makes it easier to drag to that corner
+    await page
+      .getByTestId('btn-iris-grid-settings-button-table')
+      .nth(1)
+      .click();
+
+    const table = page.locator('.lm_tabs').nth(2).locator('.lm_tab').first();
```

Comment:
Can you get the tab by the table name instead?

Reply:
`page.locator('.lm_tab').filter({ has: page.getByText('test-z') });` is what I did in the golden-layout test.

Reply:
Was thinking about this as well but the way tables are generated in the table tests give them unique and random names each time. I can rewrite it to force a table name onto the ones I created but I assume there's a reason we wanted unique names for all the tables created here?

<img width="514" alt="image" src="https://github.com/deephaven/web-client-ui/assets/55671206/eeafb4b7-1543-4f63-9325-c7919cd9deb4">

Reply:
You can create the random table name ahead of time to pass into the function and use that